files/image_config/platform/rc.local

```diff
@@ -182,7 +182,7 @@ value_extract() {
 
 program_console_speed()
 {
-    speed=$(cat /proc/cmdline | grep -Eo 'console=ttyS[0-9]+,[0-9]+' | cut -d "," -f2)
+    speed=$(cat /proc/cmdline | grep -Eo 'console=tty[a-zA-Z]+[0-9]+,[0-9]+' | cut -d "," -f2)
```

Comment:
change to tty(S|AMA)[0-9]? to be more specific?

Reply:
ok